exasol_advanced_analytics_framework/context_wrapper/context_wrapper_base.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC
+
+
+class ContextWrapperBase(ABC):
```

Comment:
We need to rename the ContextWrapper, because it should be an abstraction for the results of the return query. We want to hide the UDF specifics, because we might want to implement the EventLoop differently in the future, so we should reduce the interface to the minimum necessary and as such also adapt its name. I suggest, we call it EventContext, because it contains the Context for the current Event. I would suggest it defines the same methods as the ResultSet in https://github.com/exasol/data-science-utils-python/blob/45283be1e0a61894249cef2d2bbb4e4c4c16b2fb/exasol_data_science_utils_python/udf_utils/sql_executor.py#L5. The ResultSet doesn't provide a reset, because this is usually not supported by our clients and the UDFs are one of the rare examples, where we support this.